bigframes/dataframe.py

```diff
@@ -4574,4 +4575,12 @@ def _throw_if_null_index(self, opname: str):
 
     @property
     def semantics(self):
+        warnings.warn(
+            "The 'semantic' property will be removed. Please use 'ai' instead.",
+            FutureWarning,
+        )
         return bigframes.operations.semantics.Semantics(self)
+
+    @property
+    def ai(self):
```

Comment:
Does this need a docstring? Also, please add the `AI` class to the accessors section in the docs:

https://github.com/googleapis/python-bigquery-dataframes/blob/main/docs/reference/bigframes.pandas/frame.rst#accessors

as well as thhe table of contents

https://github.com/googleapis/python-bigquery-dataframes/blob/578081e978f2cca21ddae8b3ee371972ba723777/docs/templates/toc.yml#L44

Reply:
Done. https://screenshot.googleplex.com/3WZiTPMhKvzB6PW